pq-crypto/bike_r2/sampling.h

```diff
@@ -21,8 +22,7 @@ typedef enum
 _INLINE_ ret_t
 get_seeds(OUT seeds_t *seeds)
 {
-  // pq-random uses 0 for success and negative values for failure
-  if(get_random_bytes(seeds->seed[0].raw, sizeof(seeds_t)) == 0)
+  if(s2n_result_is_ok(get_random_bytes(seeds->seed[0].raw, sizeof(seeds_t))))
```

Comment:
You could have a macro for this.

Reply:
We actually do: `GUARD_AS_POSIX`. I was wanting to change as little on the callers of utils for now, so I'll definitely clean this up when I get to pq-crypto